content/lang/fr/404.md

```diff
@@ -1,5 +1,5 @@
-# Error
+# Erreur
 
-Yikes, looks like that page vanished.
+Yikes, ressemble à cette page disparue.
 
-Time to head [Home](/).
+Le temps de diriger [Accueil] (/).
```

Comment:
lol yeah this one seems bad on second thought - "the time" to "head" (as in, the body part) haha